### PR TITLE
Add Mobile main menu and about section

### PR DESCRIPTION
**Pull request by Basir-Mohammadi**
**Here we:**
**-Add Capstone project for the first module.**
**-Add Home and About page.**
**-Add Mobile Menu for the mobile version.**
**-Add objects and Array to make the page dynamic.**
https://www.loom.com/share/70ffe8ad544e4c3c91848b0d153ace2e

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (ShadyShawkat) [inline README.md]: - [x] Kindly edit the "Acknowledgements" section to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  </div>
+</nav>
+<!-- END OF NAVBAR --> 
+
+<!--SECTION ONE STARTS-->
+<section class="container container-intro">
+  <div class="project-head-title">
+    <h3 class="project-title">"Hello! Sharing world"</h3>
+    <h2 class="project-desc">Creative Commons Global Summit 2015</h2>
+  </div>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly edit this section's header text to have the appropriate text size to match the original design.

| | Your design | Original design |
| --- | --- | --- |
| Desktop | image | image |
| Mobile | image | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        <h5 class="details-image">
+           there of others across generational divides.
+        </h5>
+      </div>
+      
+    </div>
+    <div class="program-details">
+      <div class="image-icon">
+      <img src="imgs/realize.png" alt="Realize-image">
+    </div>
```
Comment (ShadyShawkat) [inline index.html:104]: - [x] Please make sure that the cards in this section on mobile view, don't take more width than they should which causes the UI to break.
- [x] Kindly adjust the left and right spacing of this section to match the original design.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+ 
+</div> 
+
+<div class="right">
+  
+</div>
+
+</div>
+
+</section>
```
Comment (ShadyShawkat) [inline index.html:172]: - [x] Kindly align the featured speakers cards to better match the original design. (You can use a grid which would make the cards align better).
- [ ] Please add the checkered background image added to the top left side of each speaker image. (You can set it as a background image and control its position and size using the `background-position` and `background-size` properties).

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+    </div>
+    <div class="menu-js">
+      
+      <button id="close-btn" type="button" class="close-btn x">
+        <span>&times;</span>
+      </button>
+    <div class="image-nav">
+      <a href="index.html"> <img class="progarm-logo" src="imgs/logo-project.png" alt=""></a>
+      <ul class="nav-menu">
+          <li ><a class="link about"  href="#">About</a></li>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly edit this link to redirect the user to the "about" page.
- [x] Please add a way to navigate from the "about" page to the "home" page on the mobile view. (You can add a navlink in the mobile menu that navigates to the home page)

- [x] Kindly adjust the size of the mobile menu to take 100% of the screen's height, to make your project more responsive. 
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+    </nav>
+    <!-- END OF NAVBAR --> 
+
+    <!-- FIRST SECTION-->
+    <section class="container container-intro-about">
+        <div class="project-head-title-about">
+          <h3 class="project-title-about">"Hello! Sharing world"</h3>
+          <h2 class="project-desc-about">Creative commons global summit 2015</h2>
+        </div>
+
```
Comment (ShadyShawkat) [inline about.html]: - [x] Kindly edit this section's header text to have the appropriate size and spacing around it to match the original design.

| | Your design | Original design |
| --- | --- | --- |
| Desktop | image | image |
| Mobile | image | image |

File: README.md [hunk 7]
```diff
@@ -1 +1,69 @@
-# Capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My_Portfolio
+
```
Comment (ShadyShawkat) [inline README.md]: - [x] Please change the project's title to better suit your project.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @basirking, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Capstone Approved! 🏁🏆🎉

## Hi @basirking 👋

In my opinion, Your project is complete! ✔️ 🏁 and ready for the presentation!.

Finally, **you did it**! 🎉 👏. You have completed your first 1️⃣ 🥇 cpastone in microverse. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next module. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ descriptive commit messages 
- ✔️ descriptive PR title and description
- ✔️ no linter errors
- ✔️ implemented the previous review requested changes
- The code is top-notch! 💯 and has a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**